example/spec/parallel/ingestGranule/IngestGranuleSuccessSpec.js

```diff
@@ -970,9 +1095,12 @@ describe('The S3 Ingest Granules workflow', () => {
         let file;
         let destinationKey;
         let destinations;
+        let subTestSetupError;
```

Comment:
I believe `subTestSetupError` is defined in the outer `describe` on line 819
https://github.com/nasa/cumulus/pull/2260/files#diff-5decbba358493e90d4dd7ad765c77250e8288653bb23e36a4b5e8b7fe5c1f82fR819